### ISSUE TITLE
Documentation says default seed size is 27 but log and help say 24

### ISSUE THREAD

**Issue by eboyden** [amplab/snap#149]
Using SNAP 2.0.0.  Both the long manual and the quick start guide say the default seed size is 27.  But using default indexing parameters, the aligner log file shows:
```
Welcome to SNAP version 2.0.0.

Loading index from directory... 25s.  3110877063 bases, seed size 24
Aligning.
```
And the index command help message says:
```
/tools/snap/v2.0.0/snap-aligner index
Welcome to SNAP version 2.0.0.

Usage: snap-aligner index <input.fa> <output-dir> [<options>]
Options:
 -s                Seed size (default: 24)
```

**Comment by bolosky** [comment]
The manual didn't get updated.  We changed the default from 27 in 1.0 to 24 in 2.0.

From: eboyden ***@***.***>
Sent: Thursday, January 6, 2022 9:30 PM
To: amplab/snap ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [amplab/snap] Documentation says default seed size is 27 but log and help say 24 (Issue #149)


Using SNAP 2.0.0. Both the long manual and the quick start guide say the default seed size is 27. But using default indexing parameters, the aligner log file shows:

Welcome to SNAP version 2.0.0.



Loading index from directory... 25s.  3110877063 bases, seed size 24

Aligning.

And the index command help message says:

/tools/snap/v2.0.0/snap-aligner index

Welcome to SNAP version 2.0.0.



Usage: snap-aligner index <input.fa> <output-dir> [<options>]

Options:

 -s                Seed size (default: 24)

-
Reply to this email directly, view it on GitHub<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Famplab%2Fsnap%2Fissues%2F149&data=04%7C01%7Cbolosky%40microsoft.com%7C61e00350d3dc48b3ec1908d9d19eb954%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637771301917160751%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=RVM%2Ba6EW5UfnaCZehXBZTYV0sYddz%2F4e7UBmkpiCSjY%3D&reserved=0>, or unsubscribe<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAAHPTWPZHDQNYUWCZVB5WZLUUZ24ZANCNFSM5LOAVZ4A&data=04%7C01%7Cbolosky%40microsoft.com%7C61e00350d3dc48b3ec1908d9d19eb954%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637771301917211570%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=lnn0lODvhCEF9QLaNA36aETyy%2Fdu95CujuhK6AOtYuY%3D&reserved=0>.
Triage notifications on the go with GitHub Mobile for iOS<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fapps.apple.com%2Fapp%2Fapple-store%2Fid1477376905%3Fct%3Dnotification-email%26mt%3D8%26pt%3D524675&data=04%7C01%7Cbolosky%40microsoft.com%7C61e00350d3dc48b3ec1908d9d19eb954%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637771301917211570%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=x0GDXR%2Bh2C19Gu6C8YP48gNCA9HpTtDsxTSEfUgvApM%3D&reserved=0> or Android<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fplay.google.com%2Fstore%2Fapps%2Fdetails%3Fid%3Dcom.github.android%26referrer%3Dutm_campaign%253Dnotification-email%2526utm_medium%253Demail%2526utm_source%253Dgithub&data=04%7C01%7Cbolosky%40microsoft.com%7C61e00350d3dc48b3ec1908d9d19eb954%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637771301917221513%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=psiSwxK5kA5%2FIKUP7sIUoAZTYvizxixbjlw3GXGsrzg%3D&reserved=0>.

**Comment by eboyden** [comment]
Yup I figured that's what happened.  Just letting you know the documentation needs updating.

**Comment by bolosky** [comment]
Fixed in 2.0.1
